piperider_cli/cloud/__init__.py

```diff
@@ -181,15 +182,18 @@ def get_default_project(self) -> PipeRiderProject:
         if response.status_code != 200:
             return None
 
-        for project in response.json():
-            if project.get('id'):
-                # Legacy projects api response
-                if project.get('is_default'):
-                    return PipeRiderProject(project)
-            else:
-                # New projects api response
-                for p in project.get('projects', [])[:1]:
-                    return PipeRiderProject(p)
+        responseData = response.json()
+
+        def genProjects():
```

Comment:
Is it a typo of `getProjects`?

Reply:
It is generator, so it means "GENerate projects"